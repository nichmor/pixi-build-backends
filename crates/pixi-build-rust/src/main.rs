mod build_script;
mod config;
mod protocol;
mod rust;

use protocol::RustBackendInstantiator;

#[tokio::main]
pub async fn main() {
    if let Err(err) = pixi_build_backend::cli::main(RustBackendInstantiator::new).await {
        eprintln!("{err:?}");
        std::process::exit(1);
    }
}
