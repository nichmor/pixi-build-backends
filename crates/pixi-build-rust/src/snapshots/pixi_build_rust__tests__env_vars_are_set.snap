---
source: crates/pixi-build-rust/src/main.rs
expression: generated_recipe.recipe.build.script
---
content:
  - "cargo install --locked --root \"$PREFIX\" --path . --no-track  --force"
env:
  foo: bar
